Documentation/guides/port.rst

```diff
@@ -0,0 +1,62 @@
+.. include:: /substitutions.rst
+===========================================
+How to port
+===========================================
+
+This guide explains "How to port NuttX".  At this guide, the scope of porting is to add new SoC/Board directory, the goal of porting is to comfirm to boot NuttShell(NSH).
```

Comment:
"the scope of porting is adding a new board directory, which can be built and has a working example that boots to the NuttShell (NSH)"